src/js/components/Products/Product.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react";
+import Navbar from "../Navbar/Navbar";
+import "./Product.scss";
+import  * as myConstClass from "../../constants/product-constants"
+
+class Product extends Component {
+    constructor(props){
+        super(props);
+        this.state ={
+            Name:'',
+            CoverImageLocation:'',
+            Description:'',
+            Price:'',
+            Type:'',
+            SizeOption:'',
+            ColourOption:'' ,
+            small:false,
+            medium:false,
+            large:false,
+            Speedy:false,
+            color:false,
+            kyn:false,
+            count:0,
+
+        }
+
+        this.onChange=this.onChange.bind(this);
+        this.onSubmit=this.onSubmit.bind(this);
+        this.Small=this.Small.bind(this);
+        this.Medium=this.Medium.bind(this);
+        this.Large=this.Large.bind(this);
+        this.handleChecked=this.handleChecked.bind(this);
+        this.Color=this.Color.bind(this)
+    }
+    
+    onChange(e){
+        this.setState({[e.target.name]:e.target.value});
+    }
+
+    onSubmit(e){
+        e.preventDefault();
+        const post={
+            Name:this.state.Name,
+            Description: this.state.Description,
+            Price:this.state.Price,
+            Type:this.state.Type,
+            SizeOption:this.state.count,
+            ColourOption:this.state.ColourOption,
+            CoverImageLocation:this.state.FileUpload,
+        }
+
+        fetch(myConstClass.PRODUCT_URL,{
+            method:'POST',
+            headers:{
+                'Accept': 'application/json',
+                'content-type':'application/json'
+            },
+            body:JSON.stringify(post)
+        })
+        .then(response=>response.json());
+    }
+
+    Color()
```

Comment:
Choose a specific style for your function declaration.
`Color(){}`
or 
```
Color()
{
}

```
On top of it, avoid leaving an empty line between your parentheses and your curly brackets

Reply:
i have changed the style of  function declaration  to 
color()
{
}